src/client/src/OpenFin/apps/Launcher/NlpSuggestions/executions/RfqExecution/rfqExecutionTypes.ts

```diff
@@ -0,0 +1,58 @@
+import { AckCreateRfqResponse, Direction } from "@/generated/TradingGateway"
+
+export enum NlpExecutionStatus {
```

Comment:
Should rename this aswell to differentiate it from the Trade Status, I think we don't use all of these statuses either

Reply:
we do for rfq - the loading bar shows up when its "waiting to execute"